File: SecondYear/Anantj2499_AnantJaiswal_2226cseml1088_2/Week_1/While_loop.java
import java.util.Scanner;

public class While_loop {
    public static void main(String[] args) {
        // Write your code here
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int even =0;
        int odd=0;
        int digit =0;
        while (n>0) {
            digit = n%10;
            if(digit%2==0)
                even+=digit;
            else
                odd+=digit;
            n/=10;
        }
        System.out.println(even+" "+odd);
    }
}

File: SecondYear/gautam0429_GautamAgrahari_2226ec1142_2/Week_1/While loop.cpp
#include<iostream>
using namespace std;

int main() {
	// Write your code here
	int n,r,even=0,odd=0;
	cin >> n;
	//if(n%2==0)
	while(n!=0){
		r = n%10;
		n = n / 10;
                if (r % 2 == 0) {
                  even = even + r;
                }
                else if(r%2!=0){
                    odd = odd + r;
                }
            
        }
		cout << even <<"  " << odd <<  endl;
		return 0;
}
